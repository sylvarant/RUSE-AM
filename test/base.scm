;#clo((λ x,n,y in x))
(λ  (x n y)  x)
