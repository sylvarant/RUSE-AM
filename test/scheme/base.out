#clo((Î» x,n,y in x))
